team-agreements/code-with-agreenents.md

```diff
@@ -0,0 +1,41 @@
+# Code With Agreements
+
+[CSE](../CSE.md) code with **CP (the Customer or Partner)**.
+
+Both teams from different companies gather as one team, with different backgrounds.
+This brings diversity and that's the point of the engagement, but the path may be bumpy, especially if rules are blurry or inexistent.
+It's also important to prepare for the end of the engagement, when code will still have to be maintained by CP, and/or open sourced by CSE or CP.
+
+For all these reasons, both parties need to agree on a few principles.
+
+## Tech Lead Role
+
+The Tech Lead role can be held by one person from CSE, or one person from CSE and one person from CP.
+
+When the role is played by both someone from CSE and someone from CP, they must agree on their respective scopes.
+For instance, if CP is an automotive company and the project is on Azure,
+the CSE Tech Lead will lead on Azure, and the CP Tech Lead will lead on everything related to automotive.
+
+If CP already have strong habits on software development, having the duo will be very beneficial to make sure
+these habits are both respected and challenged (if needed).
+
+## Product Owner Role
+
+The **PO (Product Owner)** role can also be held by one person from CSE, or one person from CSE and one person from CP.
+
+When the role is played by a duo, both people will work closely together to maintain a unique set of priorities
+for the backlog. When it comes to preparation of the backlog, the CP PO may have a stronger focus on what needs to be done
```

Comment:
```suggestion
for the backlog. When it comes to backlog grooming, the CP PO may have a stronger focus on what needs to be done
```

Reply:
I prefer refinement as grooming may be more about removing old stuff and give priotity to the most important one
cf https://www.agilealliance.org/glossary/backlog-grooming

Reply:
refinement works for me!